monolithic_integration_test/src/lib.rs

```diff
@@ -2,5 +2,40 @@
 
 #[cfg(feature = "daphne")]
 pub mod daphne;
-#[cfg(feature = "janus")]
 pub mod janus;
+
+lazy_static::lazy_static! {
+    static ref CONTAINER_CLIENT: testcontainers::clients::Cli =
+        testcontainers::clients::Cli::default();
```

Comment:
FYI, the Drop implementation of `Cli`'s internals is responsible for cleaning up Docker networks, so this will leak them. We could accept this, as they just represent some BoltDB rows and iptables rules. Alternately, we could create a new client object at the top of each test, or use a global `RwLock<Weak<Cli>>` with lazy initialization.

Reply:
I think you're correct, but I'm going to leave this cleanup for now. `Container` instances include an internal reference to the client that created them, so I think I'd need to do some refactoring to make the lifetimes work out.